presto-spi/src/main/java/com/facebook/presto/spi/ConnectorViewDefinition.java

```diff
@@ -22,12 +22,14 @@
     private final SchemaTableName name;
     private final Optional<String> owner;
     private final String viewData;
+    private final Optional<String> comment;
 
-    public ConnectorViewDefinition(SchemaTableName name, Optional<String> owner, String viewData)
+    public ConnectorViewDefinition(SchemaTableName name, Optional<String> owner, String viewData, Optional<String> comment)
```

Comment:
This change seems to better fit with the next diff.

Reply:
This is the correct diff.